.vscode/extensions.json

```diff
@@ -1,35 +1,41 @@
 // ⚠ Disclaimer: The typeshed team doesn't commit to maintaining this file. It exists purely for your ease of use.
-
 // Keep in alphabetical order
 {
-  "recommendations": [
-    "bierner.github-markdown-preview",
-    "bungcip.better-toml",
-    "editorconfig.editorconfig",
-    "ms-python.black-formatter",
-    "ms-python.flake8",
-    "ms-python.isort",
-    "ms-python.python",
-    "ms-python.vscode-pylance",
-    "redhat.vscode-yaml",
-],
-"unwantedRecommendations": [
-    /*
-     * Don't recommend by default for this workspace
-    */
-    "christian-kohler.npm-intellisense",
-    /*
-     * Must disable in this workspace
-     * https://github.com/microsoft/vscode/issues/40239
-    */
-    // We use black
-    "ms-python.autopep8",
-    // Not using pylint
-    "ms-python.pylint",
-    // VSCode has implemented an optimized version
-    "coenraads.bracket-pair-colorizer",
-    "coenraads.bracket-pair-colorizer-2",
-    // Obsoleted by Pylance
-    "ms-pyright.pyright",
-  ],
+    "recommendations": [
```

Comment:
Lots of whitespace changes because apparently last time I didn't "format on save" this file. This respects the configuration in `.editorconfig` and `.vscode/settigns.json`.